.rubocop.yml

```diff
@@ -109,16 +109,11 @@ Rails/Exit:
 # https://docs.rubocop.org/rubocop-rspec/cops_rspec.html#rspecfilepath
 RSpec/FilePath:
```

Comment:
When I was playing with this one locally, I just did `Enabled: false`. It should just throw an error when v3 hits and is depreciated